sunpy/util/config.py

```diff
@@ -24,40 +36,59 @@ def load_config():
     # Specify the working directory as a default so that the user's home
     # directory can be located in an OS-independent manner
     if not config.has_option('general', 'working_dir'):
-        config.set('general', 'working_dir', os.path.join(_get_home(), "sunpy"))
+        config.set('general', 'working_dir', str(Path.home() / "sunpy"))
 
     # Specify the database url as a default so that the user's home
     # directory can be located in an OS-independent manner
     if not config.has_option('database', 'url'):
-        config.set('database', 'url', "sqlite:///" + os.path.join(
-            _get_home(), "sunpy/sunpydb.sqlite"))
+        config.set('database', 'url', "sqlite:///" + str(Path.home() / "sunpy" / "sunpydb.sqlite"))
 
-    # Use absolute filepaths and adjust OS-dependent paths as needed
-    filepaths = [
-        ('downloads', 'download_dir'),
-        ('downloads', 'sample_dir')
-    ]
-    _fix_filepaths(config, filepaths)
+    working_dir = Path(config.get('general', 'working_dir'))
+    sample_dir = Path(config.get('downloads', 'sample_dir'))
+    download_dir = Path(config.get('downloads', 'download_dir'))
+    config.set('downloads', 'sample_dir', str((working_dir / sample_dir).expanduser().resolve()))
+    config.set('downloads', 'download_dir', str((working_dir / download_dir).expanduser().resolve()))
 
     return config
 
 
+def _find_config_files():
+    """Finds locations of SunPy configuration files"""
+    config_files = []
+    config_filename = 'sunpyrc'
+
+    # find default configuration file
+    module_dir = Path(sunpy.__file__).parent
+    config_files.append(str(module_dir / 'data' / 'sunpyrc'))
```

Comment:
I believe this works:
```suggestion
    config_files.append(f"{module_dir / 'data' / 'sunpyrc'}")
```

Reply:
Similarly in other places.

Reply:
I mean it works fine, it feels a little overkill to str something?

Reply:
i think, it work good,very little change f over str.

Reply:
There is also `Path.as_posix()` for a third option.

I think `str()` is probably lower overhead on the interpreter and a little more familiar to people, but I don't really think it makes any real difference.